plugin/src/withSentry.ts

```diff
@@ -1,5 +1,6 @@
-import { ConfigPlugin, createRunOncePlugin, WarningAggregator } from '@expo/config-plugins';
-import { ExpoConfig, PublishHook } from '@expo/config-types';
+import { ConfigPlugin, createRunOncePlugin, WarningAggregator } from 'expo/config-plugins';
+import type { ExpoConfig } from 'expo/config';
+import type { PublishHook } from '@expo/config-types';
```

Comment:
`PublishHook` doesn't exist in the re-exports from what I can see. It's not in the exported `d.ts` files though, so doesn't hurt (I moved `@expo/config-types` to `devDependencies`)